nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -292,6 +305,74 @@ constexpr int blockage_test_ongoing = 2;
 constexpr int max_odometry_hz = 50;
 
+struct FieldInfo
+{
+  FieldInfo(
+    bool min_value_available, bool max_value_available, bool resolution_available, float min_value,
+    float max_value, float resolution)
+  : min_value_available(min_value_available),
+    max_value_available(max_value_available),
+    resolution_available(resolution_available),
+    min_value(min_value),
+    max_value(max_value),
+    resolution(resolution)
+  {
+  }
+  bool min_value_available;
+  bool max_value_available;
+  bool resolution_available;
+  float min_value;
+  float max_value;
+  float resolution;
+};
+
+// Detection field infos
+const FieldInfo azimuth_info{true, true, false, -M_PI, M_PI, 0.f};
+const FieldInfo azimuth_std_info{true, true, false, 0.f, 1.f, 0.f};
+const FieldInfo elevation_info{true, true, false, -M_PI, M_PI, 0.f};
+const FieldInfo elevation_std_info{true, true, false, 0.f, 1.f, 0.f};
+
+const FieldInfo range_info{true, true, false, 0.f, 301.f, 0.f};
+const FieldInfo range_std_info{true, true, false, 0.f, 1.f, 0.f};
+const FieldInfo range_rate_info{true, true, false, -100.f, 100.f, 0.f};
+const FieldInfo range_rate_std_info{true, true, false, 0.f, 1.f, 0.f};
+
+const FieldInfo rcs_info{true, true, true, -128.f, 127.f, 1.f};
+const FieldInfo measurement_id_info{true, true, true, 0.f, 65535.f, 1.f};
+const FieldInfo positive_predictive_value_info{true, true, true, 0.f, 100.f, 1.f};
+const FieldInfo classification_info{true, true, true, 0.f, 255.f, 1.f};
+const FieldInfo multi_target_probability_info{true, true, true, 0.f, 1.f, 0.01f};
+const FieldInfo object_id_info{true, true, false, 0.f, 65535.f, 1.f};
+const FieldInfo ambiguity_flag_info{true, true, true, 0.f, 1.f, 0.01f};
+
+// Object field infos
+const FieldInfo age_info{true, true, true, 0.f, 65535.f, 1.f};
+const FieldInfo measurement_status_info{true, true, true, 0.f, 255.f, 1.f};
+const FieldInfo movement_status_info{true, true, true, 0.f, 255.f, 1.f};
+
+const FieldInfo position_x_info{true, true, false, -1600.f, 1600.f, 0.f};
+const FieldInfo position_y_info{true, true, false, -1600.f, 1600.f, 0.f};
+const FieldInfo position_z_info{true, true, false, -1600.f, 1600.f, 0.f};
+
+const FieldInfo velocity_x_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo velocity_y_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo velocity_z_info{false, false, false, 0.f, 0.f, 0.f};
+
+const FieldInfo acceleration_x_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo acceleration_y_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo acceleration_z_info{false, false, false, 0.f, 0.f, 0.f};
+
+const FieldInfo shape_x_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo shape_y_info{false, false, false, 0.f, 0.f, 0.f};
+const FieldInfo shape_z_info{false, false, false, 0.f, 0.f, 0.f};
+
+const FieldInfo orientation_info{true, true, false, -M_PI, M_PI, 0.f};
+const FieldInfo orientation_std_info{false, false, false, 0.f, 0.f, 0.f};
```

Comment:
My datasheet specifies `min=0` (`max=TBD`) but min is set to not available here.

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/b9fd4f49ef0404454b7dcccf472217c035f6e7d0